golang/pkg/builder/container/container_builder.go

```diff
@@ -64,7 +66,7 @@ type DockerContainerBuilder struct {
 	ctx             context.Context
 	client          client.APIClient
 	containerID     *string
-	networkIDs      []string
+	networkMap      map[string]string
```

Comment:
Does this change affect any other module?

Reply:
I am not aware, this was internal, no usage outside of the package, the only use of the change is to better inform the user about what is happening.